src/series.rs

```diff
@@ -104,15 +104,15 @@ struct DerivedProps {
     svg_elements: Vec<Html>,
 }
 
-pub struct HorizontalSeries {
+pub struct Series {
     derived_props: DerivedProps,
     _resize_listener: EventListener,
     svg: NodeRef,
 }
 
-impl HorizontalSeries {
+impl Series {
     fn derive_props(props: &Props) -> DerivedProps {
-        let classes = classes!("horizontal-series", props.name.to_owned());
+        let classes = classes!("series", props.name.to_owned());
 
         let x_scale = props.width as f32;
         let y_scale = props.height as f32;
```

Comment:
`props.width` and `props.height` are already f32's.

Reply:
Good catch. There's a few of them as a hangover from a previous commit. I'll address them now.